### ISSUE TITLE
Hook up search bars on collections show page

### ISSUE THREAD

**Issue by val99erie** [curationexperts/alexandria-v2#62]
There are 2 search bars on the collection show page:
1. The main catalog search bar in the header
2. The "Search within this collection" search bar

* [x] They need to be hooked up to correctly search, either constrained to the current collection or not.  This depends on story #61.
* [x] Make sure facets still work properly if you fix the search bars

**Comment by val99erie** [comment]
Can we use that 'cq' vs 'q' search fields to differentiate between a in-collection search and a general search?

**Comment by val99erie** [comment]
We might just need to use separate view partials for each search bar.

**Comment by mark-dce** [comment]
Would it be reasonable to remove the "Search within this collection" option for the initial release and add this or another story to the discovery backlog?  Or we can fix this to make it work now.
